pkg/machines/actions.es6

```diff
@@ -28,12 +28,17 @@ import { virt } from './provider.es6';
  */
 
 // --- Provider actions -----------------------------------------
+export function initialize() {
+    return virt('INITIALIZE');
+}
+
 /**
  *
  * @param connectionName optional - if `undefined` then for all connections
+ * @param libvirtServiceName
  */
-export function getAllVms(connectionName) {
-    return virt('GET_ALL_VMS', { connectionName });
+export function getAllVms(connectionName, libvirtServiceName) {
```

Comment:
I don't like adding the param here unless really necessarily.
It has nothing to do with the actual read of VMs.

Reply:
We decided it is fine for now